docs/user_guide/index.rst

```diff
@@ -7,6 +7,101 @@ User Guide
 
    operations/index
    gui/index
-   python/index
    reconstructions/index
 
+
+Mantid Imaging User Guide: Example Reconstruction
+-------------------------------------------------
+
+Written: November 2020
+
+Author: Arianna Wintle (STFC)
+
+Please email any suggestions to mantidimagingsupport@stfc365.onmicrosoft.com
+
+Loading Sample Stack
+####################
+
+Firstly, let's load our projections. Go to "File" then "Load" or alternatively use the shortcut "Ctrl"+"o". This brings up the following loading screen:
+
+.. image:: ../_static/loading_screen.png
+    :alt: Loading screen
+    :width: 70%
+    :align: center
+
+
+#. Click select on the sample row and go to the directory you want to load. We'll be working with the "Tilt 4 pre reco" data set.
+#. Open one of the images with the prefix "Tomo".
+#. All images with the prefix "Tomo" in the same area will be loaded
+
+Mantid Imaging should at this stage search the directory for all the files with prefixes like "Flat" and "Dark" and put them in the correct category. If this is missed or simply stored in a different location, the above steps can be repeated to find and load these.
+
+At this stage there is also the option to:
+
+#. Open only a portion of the sample images. This can be done by manually selecting the file indices or using the "Preview" Button which will set the file indices to a suitable selection.
+#. Select the pixel size of the detector used in microns. This can also be changed at a later stage.
+#. Untick a category of files to not load. For example, if both "Dark Before" and "Dark After" images have been found this can be adjusted to only one.
+
+Project Window
+##############
+
+
+.. image:: ../_static/project_view_guide.png
+    :alt: View of the application window
+    :align: center
+
+
+Operations
+##########
+
+Next let's take the sample we loaded and let's tidy it up with operations. To open the operations go to "Workflow" then "Operations".
+
+1. **Remove Outliers** will be the first operation we will apply. This has to be completed in 2 modes "Bright" and "Dark". We'll complete this for both modes with difference set to 1000 and median kernel set to size 3. Apply this to all stacks.
```

Comment:
Yeah during that discussion we found out that they use ImageJ's 50, which is equivalent to 50 for us. I checked that afterwards and the confusion seems to be because we show all pixels affected in a binary mask, while ImageJ doesn't use a binary mask, so you can only see the brightest affected pixels.

Really it's very based on the data, and the Tilt 4 is already pre-cleaned in ImageJ with 50, so with 1000 there might not be a lot left to change